code/game/objects/items/eightball.dm

```diff
@@ -41,6 +41,13 @@
 		new /obj/item/toy/eightball/haunted(get_turf(src))
 		qdel(src)
 
+/obj/item/toy/eightball/examine(mob/user)
+	..()
+	if(on_cooldown)
```

Comment:
I feel like it should display a message on use and not examine about the cooldown, it's better and more intuitive than on examine.